package/cifs/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:16.04
+RUN apt-get update && \
+    apt-get install -y jq curl nfs-common
```

Comment:
To make sure I'm on the same page, you installed `cifs-utils` package on the host and the storage plugin functions, correct?

We typically try to ship all dependencies within the Docker image so the user doesn't need to perform any manual setup on the underlying hosts. In some scenarios (RancherOS, Atomic), installing deps on the host is not even possible. I would like to see the package added, thanks.

cc @moelsayed